Library/Formula/emp.rb

```diff
@@ -0,0 +1,39 @@
+require "language/go"
+
+class Emp < Formula
+  desc "CLI for Empire."
+  homepage "https://github.com/remind101/empire"
+  url "https://github.com/remind101/empire/archive/v0.10.0.tar.gz"
+  sha256 "0f49543e25d44a05522f8e2cbb4afbb5b08153767620e304e371e13b8f574508"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    mkdir_p buildpath/"src/github.com/remind101/"
+    ln_s buildpath, buildpath/"src/github.com/remind101/empire"
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    system "go", "build", "-o", bin/"emp", "./src/github.com/remind101/empire/cmd/emp"
+  end
+
+  test do
+    require "webrick"
+
+    server = WEBrick::HTTPServer.new :Port => 8035
+    server.mount_proc "/apps/foo/releases" do |_req, res|
+      res.body = '[{"created_at":"2015-10-12T0:00:00.00000000-00:00","description":"my awesome release","id":"v1","user":{"id":"zab","email":"zab@waba.com"},"version":1}]'
```

Comment:
Any way we can break this onto two lines so it doesn't stretch the formula out so much?